AmericanMedicalAssociation/ama-style-guide-2
# [EWL-6262] Subcat Stacking Bug

## Ticket(s)
**Jira Ticket**
- [EWL-6262 | Subcat Stacking Bug](https://issues.ama-assn.org/browse/EWL-6262)

## Description
Corrects the spacing or individual subcategory items in the subcategory exploration component so that they are evenly spaced and left aligned. Cleans up logic for determining number of subcat items per row.


## To Test
- Pull `bugfix/EWL-6262-subcat-stacking-bug` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- This work affects visuals in D8 and should be tested against it to ensure there are no regressions. Update ama-d8 branch and local provision vars accordingly. 
- Go to any category page with more than one row of subcat items, such as [Delivering Care](http://ama-one.local/delivering-care) and confirm that when clicking view more, the subcategory items show left aligned when less than max possible items are on a line.
- Test this in IE 11, Safari, and Firefox to ensure that with updates everything works same as before.

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/bugfix/EWL-6262-subcat-stacking-bug/html_report/index.html).


## Relevant Screenshots/GIFs
Screenshot from Drupal



## Remaining Tasks
N/A


## Additional Notes
N/A
